twitter_sanity/scraping/basic_tweet_scraping.py

```diff
@@ -1,4 +1,6 @@
-!pip3 install twint
+git clone https://github.com/twintproject/twint.git
+cd twint
+pip3 install . -r requirements.txt
```

Comment:
You can't add all this in the python file. I meant, add twint=={version_name} as a dependency in the requirements.txt file that I've already created.